server/src/lib/hallway.ts

```diff
@@ -52,4 +52,11 @@ export class Hallway {
       this.adjacent.set(hall.id, hall);
     }
   }
+
+  public isAdjacent(testHallway: Hallway): boolean {
+    if (this.adjacent.has(testHallway.id)){
```

Comment:
Map.has returns a boolean.  We just return that.  We don't need an if statement here.